lib/puppet/parser/functions/consul_sorted_json.rb

```diff
@@ -6,11 +6,13 @@ class << self
 
     def sorted_generate(obj)
       case obj
-        when Fixnum, Float, TrueClass, FalseClass, NilClass
-          return obj.to_json
+        when NilClass
+          return "null"
```

Comment:
Why?

    $ ruby -rjson -e 'puts nil.to_json'
    null

This is what .to_json gives back on nil-